### DISCUSSION TITLE
file tags?

### DISCUSSION THREAD

**Discussion by liquidaty** [Studio-42/elFinder#3338] [Q&A]
Hi! Is there any way to "tag" a file? If not, is this something you might consider to add as a feature?

**Comment by nao-pon** [comment]
@liquidaty The "tag" feature is not available in elFinder. Also, there are no plans to implement it as standard.

However, you can create your own connector to implement its functionality and use it in elFinder. In that case, in addition to implementing the connector, it is also necessary to implement the UI side command of elFinder. (elFinder can be configured to accept its own commands)
